cot/src/middleware.rs

```diff
@@ -284,17 +285,116 @@ type LiveReloadLayerType = tower::util::Either<
     tower::layer::util::Identity,
 >;
 
+/// A middleware providing live reloading functionality.
+///
+/// This is useful for development, where you want to see the effects of
+/// changing your code as quickly as possible. Note that you still need to
+/// compile and rerun your project, so it is recommended to combine this
+/// middleware with something like [bacon](https://dystroy.org/bacon/).
+///
+/// This works by serving an additional endpoint that is long-polled in a
+/// JavaScript snipped that it injected into the usual response from your
```

Comment:
I assume that's what you meant
```suggestion
/// JavaScript snippet that it injected into the usual response from your
```

Reply:
Thanks!